### PR TITLE
veiw single blogs

### LINKED ISSUE #5: Open news on new click
image

### PR DESCRIPTION
**Pull request by promisemukundu**

### CODE REVIEW

File: src/components/Home/LatestBlog.jsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import calendar from "../../assets/img/icon/calendar.png";
 
 const LatestBlog = (props) => {
   const [news, setNews] = useState([]);
+ 
+
+
```
Comment (innoflash) [inline src/components/Home/LatestBlog.jsx]: What`s this space all about, remove this unused space

File: src/pages/Pages/BlogDetails.jsx [hunk 2]
```diff
@@ -1,7 +1,99 @@
...
+const [news, setNews] = useState();
+const params = useParams()
+
+const id = params
+
+
+
+console.log(params);
+
+
```
Comment (innoflash) [inline src/pages/Pages/BlogDetails.jsx]: Needs removing

File: src/pages/Pages/BlogDetails.jsx [hunk 3]
```diff
@@ -1,7 +1,99 @@
...
+        console.log(response);
+        setNews(response.data.articles[params.id]);
+      });
+  }, []);
+
+  
+ 
+  console.log(news);
+  console.log('kkkkkk');
+
```
Comment (innoflash) [inline src/pages/Pages/BlogDetails.jsx]: Needs removing

**Review by innoflash — CHANGES_REQUESTED** [review]

File: src/pages/Pages/BlogDetails.jsx (hunk 3 above)
Comment (promisemukundu) [reply src/pages/Pages/BlogDetails.jsx]: okay sir

**Comment by promisemukundu** [comment]
did all changes

**Review by innoflash — APPROVED** [review]

**Pull request merged**